integration/runtime_handler_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func TestRuntimeHanlder(t *testing.T) {
+	t.Logf("Create a sandbox")
```

Comment:
By default, `runtimeHandler` is not set in the integration test. Have you manually tested this?

Reply:
Here is the test result manually tested on my local machine:

```
haiyanmeng@haiyanmeng:cri$ cat /etc/containerd/config.toml 
[plugins.cri.containerd.runtimes.runc]
      # runtime_type is the runtime type to use in containerd e.g. io.containerd.runtime.v1.linux
      runtime_type = "io.containerd.runc.v1"

haiyanmeng@haiyanmeng:cri$ make test-integration RUNTIME=runc FOCUS=TestRuntimeHandler
=== RUN   TestRuntimeHandler
--- PASS: TestRuntimeHandler (0.68s)
    runtime_handler_test.go:28: Create a sandbox
    runtime_handler_test.go:30: the --runtime-handler flag value is: runc
    runtime_handler_test.go:39: Verify runtimeService.PodSandboxStatus sets RuntimeHandler
    runtime_handler_test.go:43: runtimeService.PodSandboxStatus sets RuntimeHandler to runc
    runtime_handler_test.go:45: Verify runtimeService.ListPodSandbox sets RuntimeHandler
    runtime_handler_test.go:49: runtimeService.ListPodSandbox sets RuntimeHandler to runc
PASS
```